lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2239,7 +2239,9 @@ def scatter(self, xs, ys, zs=0, zdir='z', s=20, c=None, depthshade=True,
             For more details see the *c* argument of `~.axes.Axes.scatter`.
         depthshade : bool, optional, default: True
             Whether to shade the scatter markers to give the appearance of
-            depth.
+            depth. Depthshading is performed only within the given dataset.
+            At this time we do not know about other data in the plot so that
```

Comment:
the wording is a bit awkward.

perhaps "Whether to shade... Note that each call to scatter() will perform its depthshading independently."

Reply:
:+1: Thanks, that sounds better! Fixed.